common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -91,6 +91,14 @@ public static SharedPreferencesFileManager getSharedPreferences(final Context co
         return cachedFileManager;
     }
 
+    /**
+     * This method clears the singleton cache in use by the system, in the case that unsafe operations
+     * have been performed on disk and the actual data needs to be removed.
+     */
+    public static void clearSingletonCache() {
```

Comment:
Where does this get called?

Reply:
This will get called from OneAuth side after migrateAdalCacheKey is called. I tested this locally with my Office local enlistment and clearing the objectCache seems to fix the issue of getting accounts.